client/src/components/Notices/NoticeDetail.js

```diff
@@ -39,11 +48,11 @@ const _NoticeDetail = ({notice}: NoticeDetailProps) => {
     </span>
   ) : null
 
-  const estimate = notice.price_est_low ? (
+  const estimate = notice.price_est_high !== null ? (
     <Fragment>
-      <ShowNumberCurrency num={notice.price_est_low} key="currency-1" />
+      <ShowNumberCurrency num={Math.floor(notice.price_est_low)} />
       {' - '}
-      <ShowNumberCurrency num={notice.price_est_high} key="currency-2" />
+      <ShowNumberCurrency num={Math.floor(notice.price_est_high)} />
```

Comment:
I don't want to be a nitpicker, but why `floor` instead of `round`?

Reply:
I'm more used to it, but i think you're right, `round` should be used here.